src/graphql/event/conn.rs

```diff
@@ -374,28 +374,33 @@ impl BlockListConn {
         self.inner.conn_state.clone()
     }
 
-    async fn duration(&self) -> i64 {
-        self.inner.duration
+    /// The duration is string within the range representable by a `i64`.
+    async fn duration(&self) -> StringNumber<i64> {
+        StringNumber(self.inner.duration)
     }
 
     async fn service(&self) -> String {
         self.inner.service.clone()
     }
 
-    async fn orig_bytes(&self) -> u64 {
-        self.inner.orig_bytes
+    /// The `orig bytes` in bytes within the range representable by a `u64`.
```

Comment:
`orig bytes` enclosed in backticks is not actual code. We should use a more descriptive phrase without backticks.

Reply:
The others should be corrected the same way.